docs/cinder-csi-plugin/troubleshooting.md

```diff
@@ -1,5 +1,9 @@
 # Troubleshooting
 
+## When trying to resize a Cinder Volume, PV/PVC is updated but file system not resized.
+
+Check the volume status by using `openstack volume show` command about the volume (it should have `pvc-` prefix) and see whether backend openstack need update, contact with your openstack administrator for help on the error reported.
```

Comment:
`openstack volume message list --os-volume-api-version 3.3` might also show some information about a failed resize.

e.g. `extend volume:Compute service failed to extend volume.`

We've seen this problem with live resizes. See bug https://bugs.launchpad.net/charm-cinder/+bug/1939389 and fixes https://review.opendev.org/c/openstack/charm-cinder/+/867998 and https://review.opendev.org/c/openstack/charm-cinder/+/868001.